packages/console-types/src/types/catalog/well-known-items/plugin/manifest.schema.json

```diff
@@ -464,9 +479,6 @@
                                   "type": "number"
                                 }
                               },
-                              "required": [
```

Comment:
why is port not required anymore?

Reply:
Cause not setting it has the same behaviour in Console
Also, some marketplace items have it unset so this is needed for retro-compatibility